### ISSUE TITLE
[Accessibility][DropDown]: Disabled component should have aria-disabled attribute set

### ISSUE THREAD

**Issue by prmittal** [OfficeDev/office-ui-fabric-react#1080]
For disabled components, aria-disabled attribute should be specified, else screen readers will have trouble reading it clearly. See this: http://www.w3.org/TR/wai-aria/states_and_properties#aria-disabled

Repro:
https://fabricreact.azurewebsites.net/fabric-react/master/#/examples/dropdown

The html for disabled dropdown is as below:
`<div data-is-focusable="true" id="Dropdown1755" class="ms-Dropdown is-disabled" tabindex="-1" aria-expanded="false" role="combobox" aria-label="Disabled example:" aria-activedescendant="Dropdown1755-list3" aria-controls="Dropdown1755-list"><span class="ms-Dropdown-title"><span>Option d</span></span><i class="ms-Dropdown-caretDown ms-Icon ms-Icon--ChevronDown"></i></div>`

**Comment by leddie24** [comment]
Fixed in #1231
